site/src/pages/release-notes/1.14.0.mdx

```diff
@@ -0,0 +1,337 @@
+  using an async operation. #3916 #3962
+  ```java
+  StreamMessage<Integer> userIds = StreamMessage.of(...);
+  StreamMessage<UserInfo> userInfos = streamMessage.mapAsync(this::getUserInfo);
+  
+  CompletableFuture<UserInfo> getUserInfo(int userId) {
+      ...
+  }
+  ```
+- You can now use <type://HttpMessage#peekData(Consumer)>, <type://HttpMessage#peekTrailers(Consumer)> and
+  <type://StreamMessage#peekError(Consumer)> and <type://HttpResponse#peekHeaders(Consumer)> operators on
+  <type://HttpRequest> and <type://HttpResponse>. #3097 #3988 #3949
+  ```java
+  HttpRequest request = HttpRequest.of(RequestHeaders.of(HttpMethod.POST, "/items"),
+                                       HttpData.ofUtf8("data1,data2"))
+                                       HttpHeaders.of("trailer", "foo"));
+  HttpRequest peeked = 
+    request.peekData(data -> {
+      assert data.toStringUtf8().equals("data1,data2");
+    }).peekTrailers(trailers -> {
+       assert trailers.get("trailer").equals("foo");
+    });
+  
+  HttpRequest failed = HttpRequest.ofFailure(new IllegalStateException("Something went wrong."));
+  HttpRequest peeked = failed.peekError(cause -> {
+    assert cause instanceof IllegalStateException;
+  });
+  ```
+- You can now generate an error response differently in <type://ServerErrorHandler> depending 
+  on the <type://RequestHeaders>. #4037
+- You can now use `{*var}` and `:*var` path patterns to capture rest paths of a request. #3031 #3997
+  ```java
+  Server.builder()
+        .service("/api/{item}/route/{*contents}", (ctx, req) -> {
+            // If a request path is "/api/123/route/foo/bar/baz",
+            // '*contents' should be:
+            assert ctx.pathPath("contents").equals("foo/bar/baz");
+        });
+  ```
+- You can now use <type://ServiceRequestContext#queryParams(String)> to get a decoded query part. #3955 #3960
+- You can now send query parameters using <type://QueryParams> with <type://WebClient>. #3915 #3921
+  ```java
+  QueryParams params = QueryParams.of("foo", "bar");
+  WebClient client = ...;
+  // Sends 'GET /api/items/1?foo=bar'
+  client.get("/api/items/1", params);
+  // Sends 'POST /api/items' with body '...'
+  client.post("/api/items", params, content);
+  ```
+- You can now write a `StreamMessage<HttpData>` into a file using
+  <type://StreamMessages#writeTo(StreamMessage,Path,)>. #3870 #3874
+  ```java
+  StreamMessage<HttpData> stream = ...;
+  Path destination = ...;
+  StreamMessages.writeTo(stream, destination);
+  ```
+- You can now force a state transition of a <type://CircuitBreaker>
+  by using <type://CircuitBreaker#enterState(CircuitState)>. #4010 #4022
+  ```java
+  CircuitBreaker circuitBreaker = CircuitBreaker.of(...);
+  circuitBreaker.enterState(CircuitState.FORCED_OPEN);
+  ```
+- You can now conveniently create a <type://RetryRule> and <type://RetryRuleWithContent> with their
+  specific <type://Backoff>. #3875 #2899 #3895
```

Comment:
nit; `their` makes it sound like `Backoff` should only be used with `RetryRule*`

```suggestion
- You can now conveniently create a <type://RetryRule> and <type://RetryRuleWithContent> by specifying
  a <type://Backoff>. #3875 #2899 #3895
```